libs/dh/shared/data-access-mocks/src/lib/data/esett-balance-responsible-messages.ts

```diff
@@ -89,7 +89,7 @@ export const eSettBalanceResponsibleMessages = (apibase: string): BalanceRespons
       balanceResponsible: '999',
       gridArea: '000',
       gridAreaWithName: null,
-      meteringPointType: TimeSeriesType.Production,
+      meteringPointType: BalanceResponsibilityMeteringPointType.Production,
```

Comment:
suggestion: let's remove the type casting on line 98 in order to get better type checking.